src/writeData/components/PluginCreateConfigurationCustomize.tsx

```diff
@@ -37,20 +37,49 @@ import {getDataLoaders} from 'src/dataLoaders/selectors'
 import {downloadTextFile} from 'src/shared/utils/download'
 
 // Constants
-import {TELEGRAF_DEFAULT_OUTPUT} from 'src/writeData/constants/contentTelegrafPlugins'
+// import {TELEGRAF_DEFAULT_OUTPUT} from 'src/writeData/constants/contentTelegrafPlugins'
+
+type ParamsType = {
+  [param: string]: string
+}
 
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = PluginCreateConfigurationStepProps & ReduxProps
 
 const PluginCreateConfigurationCustomizeComponent: FC<Props> = props => {
   const {
+    onAddTelegrafPlugins,
     onSetTelegrafConfigName,
     onSetTelegrafConfigDescription,
     telegrafConfigDescription,
     telegrafConfigName,
+    pluginConfig,
+    setPluginConfig,
   } = props
 
-  const workingConfig = TELEGRAF_DEFAULT_OUTPUT
+  const {contentID} = useParams<ParamsType>()
+
+  const workingConfig = pluginConfig
+  useEffect(() => {
+    import(
+      `src/writeData/components/telegrafInputPluginsConfigurationText/${contentID}.conf`
```

Comment:
can this component ever be rendered without a `contentID`? if so, this will crash spectacularly and there's no `catch`

Reply:
Good _catch_ - :grin:

Yes, both promise rejection and uncaught error handling needs to be added here. I will do so.